### ISSUE TITLE
[iOS] getPhotos `include` not working 

### ISSUE THREAD

**Issue by myleshk** [react-native-cameraroll/react-native-cameraroll#576]
The following code is not working as expected.

### Code
```
CameraRoll.getPhotos({
  first: 10,
  groupTypes: album.type,
  groupName: album.title,
  include: [
    'fileSize',
    'filename',
  ],
})
```
### Expected
It returns correct file size and filename on iOS.

### Actual
It is always broken like below, for all types of files (video, photo, in SmartAlbum or normal Album):
```
"filename": null,
"fileSize": 0,
```

### Attempts made
- Updated this package to latest v7.3.0.
- Tried on latest React Native v0.73 and an older version v0.71.
- Tried on 2 different iPhones:
  - iPhone 7 with iOS 15.8
  - iPhone 13 with iOS 17.2.1
- Latest Xcode v15.2

### Minimal reproduction
https://github.com/myleshk/react-native-cameraroll-issue

<img src="https://github.com/react-native-cameraroll/react-native-cameraroll/assets/15956924/030b37cd-f65b-48ea-a586-29a5b325696f" width=300 />
<img src="https://github.com/react-native-cameraroll/react-native-cameraroll/assets/15956924/e490a62d-d4f7-4d4c-928a-ef24c40d9145" width=300 />

**Comment by phuoctai12t** [comment]
Downgrade to 7.2.2 work for me

**Comment by ravishankar3961** [comment]
As per Apple the `PHAssetResource.fileSize` is now deprecated thats why you are getting this error. 
App is also getting rejected by Apple if you are using this library for now. 

Usage found in this library:
`NSNumber* fileSize = [resource valueForKey:@"fileSize"];`

**Comment by Arjan-Zuidema** [comment]
@ravishankar3961 do you have any links to documentation? We currently use `[resource valueForKey:@"fileSize"]` and not `PHAssetResource.fileSize`. What do you suggest we should use?

**Comment by ravishankar3961** [comment]
@Arjan-Zuidema I am not sure whether this is deprecated or become non-public. One solution found on apple forum was to use `FileManager`

`
let attr = try FileManager.default.attributesOfItem(atPath: filePath)
    fileSize = attr[FileAttributeKey.size] as! UInt64
`

**Comment by ravishankar3961** [comment]
@Arjan-Zuidema  I am not sure whether this is deprecated or become non-public. Using `FileManager` for a collection of photos seems to be a bad idea maybe. Couldn't find any new alternative as of now.

**Comment by zatteo** [comment]
It should be fixed with version 7.4.2.
